package.json

```diff
@@ -23,28 +23,28 @@
   "author": "GitHub",
   "license": "MIT",
   "dependencies": {
-    "@actions/cache": "^1.0.9",
+    "@actions/cache": "^1.0.10",
     "@actions/core": "^1.2.6",
-    "@actions/exec": "^1.0.1",
-    "@actions/io": "^1.1.0"
+    "@actions/exec": "^1.1.1",
+    "@actions/io": "^1.1.2"
```

Comment:
Doi we have any change in these toolkit packages for node16?

Reply:
These were automatically updated when I followed the guidelines for the version bump.